### ISSUE TITLE
Updating moving_variance and moving_mean

### ISSUE THREAD

**Issue by Arsenal591** [YunYang1994/tensorflow-yolov3#23]
Hello, in your code, I didn't find `moving_variance `and `moving_mean` of batch norm layer is updated during training phase. Is it a mistake, or you do it on purpose?

**Comment by Arsenal591** [comment]
Thanks.

**Comment by YunYang1994** [comment]
update it now
